terraform/Jenkinsfiles/TF_Scaffold.groovy

```diff
@@ -5,7 +5,8 @@ Map <String, Map<String, String>> componentImageBranch = [
   nhais:           [ecrRepo: "nhais",                branch: "develop"],
   nhais_responder: [ecrRepo: "nhais-fake-responder", branch: "origin-develop"],
   gp2gp:           [ecrRepo: "gp2gp",                branch: "main"],
-  mhs:             [ecrRepo: "mhs/outbound",         branch: "develop"] // mhs/route and mhs/inboud will have the same tag, no need to search in them
+  mhs:             [ecrRepo: "mhs/outbound",         branch: "develop"], // mhs/route and mhs/inboud will have the same tag, no need to search in them
+  lab-results:     [ecrRepo: "lab-results",          branch: "develop"]
```

Comment:
for new repos we use "main" which used to be "master" see: http://ec2-3-9-114-173.eu-west-2.compute.amazonaws.com/job/integration-adaptor-lab-results/job/main/

Reply:
updated lab-resuts branch name to "main"